buildLauncher.ps1

```
@@ -1,273 +1,275 @@
+    提供其他重要信息，如注意事项、限制条件、已知问题等。
+    .INPUTS
+    描述函数可以接收的输入类型，通常用于支持管道输入的函数。
+    .OUTPUTS
+    描述函数的输出类型和内容。
+    #>
+    $resPath = "ico.res";   # C++图标res资源
+    $rcPath = "ico.rc";     # C++图标rc资源
+    Set-Location -Path "musync_data/";
+    if (-not (Test-Path -Path $resPath)) {
+        & windres $rcPath -O coff -o $resPath;
+        Write-Host "图标资源 ${path}${resPath} 创建成功！" -ForegroundColor Green;
```

PSScriptAnalyzer (1.25.0)

[warning] File 'buildLauncher.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)